### PR TITLE
add tests on sendCollateral's Production contract

### PR DESCRIPTION
**Pull request by louptheron**

### CODE REVIEW

**Comment by n0izn0iz** [comment]
woops sorry pour le merge pdt que tu bosse

**Comment by n0izn0iz** [comment]
ya une fonction vide

**Comment by louptheron** [comment]
et moi des tests qui fail #merciCI

File: contracts/Production.sol [hunk 1]
```diff
@@ -18,12 +18,7 @@ contract Production {
 		SellerPaid
 	}
 
-	function () payable {
-		if (msg.value != uint(price) && state == State.RequestApproved) {
-			throw;
-		} else {
-			state = State.CollateralPaid;
-		}
+	function () {
```
Comment (n0izn0iz) [inline contracts/Production.sol:21]: empty function?

**Review by n0izn0iz — DISMISSED** [review]
Looks good except empty func

File: contracts/Production.sol (hunk 1 above)
Comment (n0izn0iz) [reply contracts/Production.sol:21]: no (solidity "fallback")

**Review by n0izn0iz — APPROVED** [review]
O! O! OK!

**Comment by n0izn0iz** [comment]
wtf, the build failed once then passed on the same code

**Comment by n0izn0iz** [comment]
@louptheron there might be uninitialized variables used

**Comment by louptheron** [comment]
Seems to be a 'testrpc' issue with gas provision on used accounts

**Comment by n0izn0iz** [comment]
yeah but why on one build and not another?

**Pull request merged**